functions/index.ts

```diff
@@ -77,7 +95,45 @@ export async function deleteGameHttpResponse(game: Game) {
   }
 }
 
-export function httpResponse(data: IHttpResponse) : {} {
+export async function getWishlistHttpResponse(userID: string) {
+  try {
+    let wishlist = new Wishlist(userID);
+    let response = await getCollection(wishlist);
+    return httpResponse({statusCode: 200, body: JSON.stringify(response)});
+  } catch (err: any) {
+    return httpResponse({statusCode: err.statusCode, body: err.message});
```

Comment:
I don't think I saw error codes being included in any of the thrown errors for Collections